content/tutorials/manage-the-rippled-server/installation/rippled-1-3-0-migration-instructions.md

```diff
@@ -0,0 +1,74 @@
+# rippled v1.3.0 Migration Instructions
+
+This document describes the migration process for upgrading from `rippled` 1.2.4 or earlier to `rippled` v1.3.0 or later. This is necessary because the `rippled` install process has changed as of version 1.3.0.
+
+This document provides migration steps for upgrading on supported platforms:
+
+- [CentOS or Red Hat Enterprise Linux (RHEL)](#migration-on-centos-or-red-hat-enterprise-linux-rhel)
+- [Ubuntu Linux](#migration-on-ubuntu-linux)
+
+For other platforms, see the updated instructions for compiling from source. ([Ubuntu](build-run-rippled-ubuntu.html), [macOS](build-run-rippled-macos.html), or [Windows](https://github.com/ripple/rippled/tree/develop/Builds/VisualStudio2017))
+
+
+## Migration on CentOS or Red Hat Enterprise Linux (RHEL)
+
+Ripple's official RPM repository and instructions for using it have changed. To migrate from the old repository to the new one, complete the following steps:
+
+1. Stop the `rippled` server.
+
+        $ sudo systemctl stop rippled.service
+
+2. Replace Ripple's old yum repo file with the new one:
```

Comment:
what do you think about an additional step here that is `rpm --erase=ripple-repo`? The old step 1 of rpm was to install that repo package which we are no longer supporting and it will probably conflict with step 2 here. This cleanup step is only relevant in the 1.2 to 1.3 upgrade and probably could be removed in future revisions.

Reply:
I'd have to test that it works as intended but I think that's a good idea. The step here overwrites the repo file that was there before (note the lack of `-a` on the `tee` command) but if you were to later remove the (old) repo package it might erase the (updated) file, so probably best to do it as you mentioned and remove the file first.

Will test this and see what happens